subsys/bluetooth/controller/hci_internal.c

```diff
@@ -1234,6 +1301,151 @@ static uint8_t le_controller_cmd_put(uint8_t const * const cmd,
+		return sdc_hci_cmd_le_big_terminate_sync(
+			(sdc_hci_cmd_le_big_terminate_sync_t const *)cmd_params,
+			(sdc_hci_cmd_le_big_terminate_sync_return_t *)event_out_params);
+#endif /* CONFIG_BT_CTLR_SYNC_ISO */
+
+#ifdef CONFIG_BT_CTLR_ADV_ISO
+	case SDC_HCI_OPCODE_CMD_LE_CREATE_BIG:
+		return sdc_hci_cmd_le_create_big((sdc_hci_cmd_le_create_big_t const *)cmd_params);
+	case SDC_HCI_OPCODE_CMD_LE_CREATE_BIG_TEST:
+		return sdc_hci_cmd_le_create_big_test(
+			(sdc_hci_cmd_le_create_big_test_t const *)cmd_params);
+	case SDC_HCI_OPCODE_CMD_LE_TERMINATE_BIG:
+		return sdc_hci_cmd_le_terminate_big(
+			(sdc_hci_cmd_le_terminate_big_t const *)cmd_params);
+#endif /* CONFIG_BT_CTLR_ADV_ISO */
+
+#ifdef CONFIG_BT_CTLR_ISO_TX_BUFFERS
+	case SDC_HCI_OPCODE_CMD_LE_READ_BUFFER_SIZE_V2:
+		*param_length_out += sizeof(sdc_hci_cmd_le_read_buffer_size_v2_return_t);
+		return sdc_hci_cmd_le_read_buffer_size_v2(
+			(sdc_hci_cmd_le_read_buffer_size_v2_return_t *)event_out_params);
+
+	case SDC_HCI_OPCODE_CMD_LE_READ_ISO_TX_SYNC:
+		*param_length_out += sizeof(sdc_hci_cmd_le_read_iso_tx_sync_return_t);
+		return sdc_hci_cmd_le_read_iso_tx_sync(
+			(sdc_hci_cmd_le_read_iso_tx_sync_t const *)cmd_params,
+			(sdc_hci_cmd_le_read_iso_tx_sync_return_t *)event_out_params);
+
+	case SDC_HCI_OPCODE_CMD_LE_ISO_TRANSMIT_TEST:
+		*param_length_out += sizeof(sdc_hci_cmd_le_iso_transmit_test_return_t);
+		return sdc_hci_cmd_le_iso_transmit_test(
+			(sdc_hci_cmd_le_iso_transmit_test_t const *)cmd_params,
+			(sdc_hci_cmd_le_iso_transmit_test_return_t *)event_out_params);
+#endif /* CONFIG_BT_CTLR_ISO_TX_BUFFERS */
+
+#if defined(CONFIG_BT_CTLR_ISO_RX_BUFFERS)
+	case SDC_HCI_OPCODE_CMD_LE_ISO_RECEIVE_TEST:
+		*param_length_out += sizeof(sdc_hci_cmd_le_iso_receive_test_return_t);
+		return sdc_hci_cmd_le_iso_receive_test(
+			(sdc_hci_cmd_le_iso_receive_test_t const *)cmd_params,
+			(sdc_hci_cmd_le_iso_receive_test_return_t *)event_out_params);
+#endif
+
+#if defined(CONFIG_BT_CTLR_ISO)
+	case SDC_HCI_OPCODE_CMD_LE_ISO_READ_TEST_COUNTERS:
+		*param_length_out += sizeof(sdc_hci_cmd_le_iso_read_test_counters_return_t);
+		return sdc_hci_cmd_le_iso_read_test_counters(
+			(sdc_hci_cmd_le_iso_read_test_counters_t const *)cmd_params,
+			(sdc_hci_cmd_le_iso_read_test_counters_return_t *)event_out_params);
+
+	case SDC_HCI_OPCODE_CMD_LE_SETUP_ISO_DATA_PATH:
+		*param_length_out += sizeof(sdc_hci_cmd_le_setup_iso_data_path_return_t);
+		return sdc_hci_cmd_le_setup_iso_data_path(
+			(sdc_hci_cmd_le_setup_iso_data_path_t const *)cmd_params,
+			(sdc_hci_cmd_le_setup_iso_data_path_return_t *)event_out_params);
+
+	case SDC_HCI_OPCODE_CMD_LE_REMOVE_ISO_DATA_PATH:
+		*param_length_out += sizeof(sdc_hci_cmd_le_remove_iso_data_path_return_t);
+		return sdc_hci_cmd_le_remove_iso_data_path(
+			(sdc_hci_cmd_le_remove_iso_data_path_t const *)cmd_params,
+			(sdc_hci_cmd_le_remove_iso_data_path_return_t *)event_out_params);
+
+	case SDC_HCI_OPCODE_CMD_LE_ISO_TEST_END:
```

Comment:
we can do that in a separate PR, but should we consider ifdef-ing out the test commands?

Reply:
I agree. We could consider adding more ifdefs for both the ISO test commands and also the Create CIG/BIG Test commands, Read Link quality and others